Formula/docuum.rb

```diff
@@ -0,0 +1,17 @@
+class Docuum < Formula
+  desc "Perform least recently used (LRU) eviction of Docker images"
+  homepage "https://github.com/stepchowfun/docuum"
+  url "https://github.com/stepchowfun/docuum/archive/v0.19.2.tar.gz"
+  sha256 "8cbf24dd9b1e8945526b32ceadb68ba44f9e60b4d89cbd04cb5574181888b533"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "USAGE", shell_output("#{bin}/docuum --help")
```

Comment:
Can we try interacting with a non-existent Docker container and check we get the expected error message and exit code?

Reply:
Hm, Docuum doesn't interact with particular user-specified containers. It looks at all the images that exist locally and deletes the ones that haven't been used recently. So I think the answer is no.

Reply:
And if there is no docker daemon running when it tries to clean up? Does it throw an error?

Reply:
In that case, Docuum will patiently wait for a Docker daemon to start (in a retry/sleep loop)—indefinitely. The reason it has this behavior is because Docuum is often run as a daemon, like Docker, and often it's ready before Docker is when a machine boots.

Would it be a better test to pass in some invalid command-line argument and check for an expected error message?

Alternatively, I could write a more comprehensive test that essentially mimics [this integration test](https://github.com/stepchowfun/docuum/blob/main/integration_test.sh), but that test would end up deleting Docker images on whatever machine happens to run it, and it would require Docker to be installed. I'm not sure exactly what environment the tests run in or how appropriate that would be (and it feels a little redundant).

Open to other ideas—let me know what you think. Thanks for brainstorming with me on this.

Reply:
We don't have Docker installed on our macOS runners, and our Linux runners already tests inside a Docker container, so we probably won't be able to do with a test that needs Docker.

Is there no way to set a timeout on Docuum waiting for a Docker daemon to start? Does it generate any logs while waiting? Can we start it, wait for it to generate logs, and then kill it and verify a substring of those logs?

Reply:
What we mostly want is that the program tries to perform it's main function. A normal codepath that fails is already way more likely to reveal issues then a codepath that just prints `version: 1.0.0`

Reply:
@SMillerDev Makes sense to me. I've implemented @carlocab's suggestion to have the test start Docuum, wait for a specific line in the logs, and then kill the subprocess.

Thanks all for your patience. I've updated this PR—let me know what you think.